### PR TITLE
Update triggers to start

### PR DESCRIPTION
**Pull request by m-goggins**
The PR adds to the deployment script to ensure that any newly created triggers associated with pipelines are "started" and will run according to their set time.

### CODE REVIEW

**Review by robertmitchellv — APPROVED** [review]
LGTM ✅ 

these triggers are going to be super helpful 👏🏻

File: scripts/Synapse/updateECRDataStore.ipynb
Comment (DanPaseltiner) [inline scripts/Synapse/updateECRDataStore.ipynb:1]: 💯

**Review by DanPaseltiner — APPROVED** [review]
LGTM. Have we tested this in Azure in anyway? For example change the trigger time to be very soon, running a deploy and confirming that everything runs as expected?

File: scripts/Synapse/updateECRDataStore.ipynb
Comment (m-goggins) [reply scripts/Synapse/updateECRDataStore.ipynb:1]: Working on it now! It _should_ kick off at 10:15am PT

Comment (DanPaseltiner) [reply scripts/Synapse/updateECRDataStore.ipynb:1]: You rock!

Comment (m-goggins) [reply scripts/Synapse/updateECRDataStore.ipynb:1]: Finally for this to working in `dev2` (on an hourly basis for testing purposes) and made the appropriate changes to that the ECR data store refresh triggers each day at 5:30pm PT and the weekly analytics pipeline triggers each week on Wednesday at 8:45pm PT.

image

**Pull request merged**